guides/source/index.html.erb

```diff
@@ -7,6 +7,10 @@
 
 <% content_for :index_section do %>
 <nav id="subCol" aria-label="Chapter" class="guide-index">
+  <a id="chapter-nav-skip-link" href="#article-body" class="skip-link">
+    Skip to article body
+  </a>
+
   <dl>
```

Comment:
This section is being read as if it were a list of items, and it's not. Let's ignore it.

```suggestion
  <dl role="presentation">
```